manim/mobject/types/dot_cloud.py

```diff
@@ -0,0 +1,44 @@
+__all__ = ["TrueDot", "DotCloud"]
+
+import numpy as np
+
+from ...constants import ORIGIN, RIGHT, UP
+from ...utils.color import YELLOW
+from .opengl_point_cloud_mobject import OpenGLPMObject
+
+
+class DotCloud(OpenGLPMObject):
+    def __init__(
+        self, color=YELLOW, stroke_width=2.0, radius=2.0, density=10, **kwargs
+    ):
+        self.radius = radius
```

Comment:
> it seems like `radius` is defined as a `_Data()` attribute in manimgl, maybe it'd be worth doing the same here?

This was another compatibility thing. I think manimce uses radius to define the cloud radius and stroke-width to define the dot radius, so to keep comptible with cairo I have maintained it to mean the same thing. Manimgl uses radius to define the dot radius from what I remember. I think tbh it would be better to have point_radius and cloud_radius to be clear but maybe we can do that after dropping cairo?

Reply:
Would it really make sense to have this class be compatible with cairo? Given that it inherits from `OpenGLPMobject` and relies on shaders, it might be better to have it be incompatible, like `OpenGLVMobject` or `OpenGLSurface` for example.